### PR TITLE
Add variables for transitions

### PR DESCRIPTION
**Pull request by tomlutzenberger**
Revision of #21334

See also #21266

### CODE REVIEW

File: scss/_buttons.scss [hunk 1]
```diff
@@ -15,7 +15,7 @@
   user-select: none;
   border: $input-btn-border-width solid transparent;
   @include button-size($btn-padding-y, $btn-padding-x, $font-size-base, $btn-border-radius);
-  @include transition(all .2s ease-in-out);
+  @include transition($transition-base);
```
Comment (Starsam80) [inline scss/_buttons.scss]: This should be its own variable: `$btn-transition`

File: scss/_images.scss [hunk 2]
```diff
@@ -16,7 +16,7 @@
   background-color: $thumbnail-bg;
   border: $thumbnail-border-width solid $thumbnail-border-color;
   @include border-radius($thumbnail-border-radius);
-  @include transition(all .2s ease-in-out);
+  @include transition($transition-base);
```
Comment (Starsam80) [inline scss/_images.scss]: This should be its own variable: `$thumbnail-transition`

**Review by Starsam80 — CHANGES_REQUESTED** [review]
You didn't have to close the old PR, `git push --force` does exist. If you fix the 2 notes I made, LGTM.

First time doing a GitHub review. No idea how this is going to work

**Comment by guylepage3** [comment]
@tomlutzenberger 
I agree with @Starsam80, you shouldn't keep moving this around from issue to issue to PR to PR. It makes it very difficult for others contributing to follow along.

Again as stated way back in issue https://github.com/twbs/bootstrap/issues/21266 & https://github.com/twbs/bootstrap/pull/21334
> I feel that the naming convention for these variables should match btn names, reducing complexity.

The reason being is that this could limit the number of variables to a standard set. Bootstrap is not created to offer "all" solutions but a couple. It's just a framework. Not a full huge behemoth framework that does everything. IMO.

**Comment by tomlutzenberger** [comment]
Yes, I have to admit that I haven't thought of the forced push.
I'll fix those 2 things.

**Review by Starsam80 — APPROVED** [review]

**Pull request merged**